packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -100,7 +103,7 @@ export class PluginLedgerConnectorQuorum
     Checks.truthy(options, `${fnTag} arg options`);
     Checks.truthy(options.rpcApiHttpHost, `${fnTag} options.rpcApiHttpHost`);
     Checks.truthy(options.instanceId, `${fnTag} options.instanceId`);
-    Checks.truthy(options.pluginRegistry, `${fnTag} options.pluginRegistry`);
+    //Checks.truthy(options.pluginRegistry, `${fnTag} options.pluginRegistry`);
```

Comment:
@TonyRowntree The plugin registry itself does not need to be optional because the API server is guaranteed to inject it to all plugin constructor options anyway (even if the options interface of a plugin doesn't actually ask for it, it's always there at runtime).